app/api/srch/search.rb

```diff
@@ -348,7 +348,7 @@ class Search < Grape::API
     def self.execute(endpoint, params)
```

Comment:
Yes for validation maybe just passing some bad data into execute() and confirming that it rejects it.